### ISSUE TITLE
Save "Dont't use %APPDATA%" in installer

### ISSUE THREAD

**Issue by da411d** [notepad-plus-plus/notepad-plus-plus#5886]
I install N++ in custom directory and without APPDATA. And in every update i need to tick this checkbox. Please save the state of this checkbox as you save the instalation path.
Also it wil be great tu auto-update N++ without interface - just click "Update" and it reopens the editor.

![Screenshot_1](https://user-images.githubusercontent.com/12462454/60710433-617c9a00-9f1b-11e9-90f8-bfc84805fab0.png)

**Comment by JimF42** [comment]
I agree about the quite auto-update. Or if an interface is desired, as the installer lets you change options, like themes, etc., how about a button on the first screen for "Express" that will skip rest of the screens as if the user just pressed "Next" until the last screen.

**Comment by sXNA** [comment]
Fixed by https://github.com/notepad-plus-plus/notepad-plus-plus/commit/a6a1442babea25e52feedf34c707cf8d72266c2e
